source/guide_postgresql.rst

```diff
@@ -251,6 +248,17 @@ Edit ``~/opt/postgresql/data/postgresql.conf`` and set the key values ``listen_a
                                         # (change requires restart)
 
 
+Because postgres defines the socket as abstract, you cannot see it directly in the filesystem.
```

Comment:
Hm, I can see the socket just fine using `ls -la`. Which utility isn't able to see it? Or is this new in more recent pg versions?

Reply:
my mistake, didnt try -la because thats usually my default anyway. gonna change the text